### DISCUSSION TITLE
Some files crash neovim with no-neck-pain open

### DISCUSSION THREAD

**Discussion by Demianeen** [shortcuts/no-neck-pain.nvim#337] [Q&A] [answered]
It is not issue because I am not sure how to reproduce it properly. It is easily reproducible for me with .zshrc file though. When I try to move upward in the top half of the zshrc file, it just crash neovim. Take a look:

https://github.com/shortcuts/no-neck-pain.nvim/assets/51330172/ec587c9d-5ad1-4afc-aef8-245aedc8b2c0

I also checked with all plugins disabled except no-neck-pain and lazyvim (with lazyvim default plugins disabled, it works fine). So, it seems there is some clash for me with lazyvim. Do you use lazyvim? If yes, have you ever had a similar issue? Or what at least can it be?

**Comment by shortcuts** [comment, marked as answer]
Hey @Demianeen thanks for reporting! I don't use lazyvim myself but I can try to install it and see if I can reproduce. Considering how many plugins there is by default I believe it can be a bit tricky to determine which one is conflicting though

Could you try to `LspStop` when you enter your zshrc and see if the issue still occurs?

**Reply by Demianeen** [reply]
Hi! I tried a few days and it just works. I will try what you say and unmark the answer if I will have any issues. Sorry, to be a pain)
